docs/source/getting-started/index.rst

```diff
@@ -78,7 +81,7 @@ In one terminal, at the root of the ``create-daml-app`` directory, run the scrip
 
     ./daml-start.sh
 
-This script is just shorthand for ``daml start`` with some arguments, which does a few things:
+You will know that script has run successfully when you see the ``INFO  com.digitalasset.http.Main$ - Started server: ServerBinding(/0:0:0:0:0:0:0:0:7575)`` message in terminal. The script is just shorthand for ``daml start`` with some arguments, which does a few things:
```

Comment:
Is  `0.0.0.0` still accurate? I thought we changed that to `127.0.0.1`.

Reply:
That's copy/pasted from terminal 🤷‍♂

Reply:
on which release? On the latest snapshot I get `Started server: ServerBinding(/127.0.0.1:7575)`

Reply:
0.13.54  (project SDK version from daml.yaml)

Reply:
Hm, I don’t quite know what the current plans are for keeping create-daml-app in sync with this repo. It seems very weird to have the docs for create-daml-app which are published as part of the SDK docs document an older version. Either way, happy to keep it like it is for now but maybe add a comment so we don’t forget to update it.

cc @hurryabit @rohanjr

Reply:
```suggestion
You will know that script has run successfully when you see the ``INFO  com.digitalasset.http.Main$ - Started server: ServerBinding(/0:0:0:0:0:0:0:0:7575)`` or ``INFO  com.digitalasset.http.Main$ - Started server: ServerBinding(/127.0.0.1:7575)`` message in terminal. The script is just shorthand for ``daml start`` with some arguments, which does a few things:
```

@cocreature would this be better?